mewarpx/tests/test_emission.py

```diff
@@ -149,9 +150,14 @@ def test_circle_emitter():
         df[label + '_std'] = np.std(res_dict[label])
 
     assert testing_util.test_df_vs_ref(
-        testname="circle_emitter", df=df, margin=0.4
+        testname="circle_emitter", df=df, margin=0.3
     )
 
+    emitter.plot_contours()
+    plotfile =  f"{cylinder.name}_contour_plot.png"
```

Comment:
Do we save plots as PNGs or JPEGs, or does it even matter? PNG's are higher quality but I feel like the difference doesn't matter for these plots.

Reply:
@MKieburtz  I don't think it matters a ton but lets stick with PNG's for the sake of consistency.

Reply:
Good question, I'm not sure if there is a standard. I just went with PNG since I saw other examples using PNG.